client/Dockerfile

```diff
@@ -2,12 +2,8 @@ FROM node:10.13.0-alpine
 
 WORKDIR /usr/app
 
-COPY package.json .
-
-COPY package-lock.json .
+COPY . /usr/app 
 
 RUN npm install --quiet
 
-ADD . /usr/app
-
```

Comment:
https://nickjanetakis.com/blog/docker-tip-2-the-difference-between-copy-and-add-in-a-dockerile
"A valid use case for ADD is when you want to extract a local tar file into a specific directory in your Docker image. This is exactly what the Alpine image does with ADD rootfs.tar.gz /."